### PR TITLE
feat(Portals): Refactor delegations queries into module

### PR DESCRIPTION
**Pull request by snaerseljan**
# Move delegations queries into to the module.

## What
This moves the queries as close as possible to the components in the delegation module. By doing that we ensure encapsulation of the module queries.

## Why

We do not want to have `service-portal` references in the `libs/portals/shared-modules/delegations` module and we would like to keep this convention on when generating graphql hooks

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/portals/shared-modules/delegations/project.json [hunk 1]
```diff
@@ -25,6 +25,13 @@
       "options": {
         "command": "yarn ts-node -P libs/localization/tsconfig.lib.json libs/localization/scripts/extract 'libs/portals/shared-modules/delegations/src/{components,screens,hooks}/**/*.{js,ts,tsx}'"
       }
+    },
+    "schemas/codegen": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "output": "libs/portals/shared-modules/delegations/src/**/*.codegen.ts",
```
Comment (eirikurn) [inline libs/portals/shared-modules/delegations/project.json:32]: Isn't the default `*.generated.ts`?

File: libs/portals/shared-modules/delegations/src/components/access/AccessConfirmModal.tsx [hunk 2]
```diff
@@ -54,8 +55,9 @@ export const AccessConfirmModal = ({
     setError(formError)
   }
 
-  const toName = delegation?.to?.name
-  const toNationalId = delegation?.to?.nationalId
+  const toName = (delegation as AuthCustomDelegationOutgoing)?.to?.name
```
Comment (eirikurn) [inline libs/portals/shared-modules/delegations/src/components/access/AccessConfirmModal.tsx:58]: Can you change the component props to use the right type if this modal is only for outgoing delegations?

**Review by eirikurn — APPROVED** [review]
Nice! Wish this required fewer type assertions, but I don't want us to spend time on it :)

File: libs/portals/shared-modules/delegations/src/components/access/AccessConfirmModal.tsx (hunk 2 above)
Comment (snaerseljan) [reply libs/portals/shared-modules/delegations/src/components/access/AccessConfirmModal.tsx]: Good catch

**Pull request merged**